docs/grover_algorithm_qi.ipynb

```diff
@@ -787,17 +826,20 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Backend: QX single-node simulator (qubits 26)\n",
-      "Backend: QX single-node simulator SurfSara (qubits 31)\n",
-      "Backend: QX multi-node simulator SurfSara (qubits 37)\n"
-     ]
+     "data": {
+      "text/plain": [
+       "[<QuantumInspireBackend('QX single-node simulator') from QuTech Delft()>,\n",
```

Comment:
It is, but now it looks worse since __repr__ is not implemented:
`[<QuantumInspireBackend('QX single-node simulator') from <quantuminspire.qiskit.quantum_inspire_provider.QuantumInspireProvider object at 0x000001B171E5AA20>()>`